app/assets/v2/css/quests.css

```diff
@@ -1555,7 +1555,7 @@
             }
             body.stage_4 #header {
                 margin-top: -2%;
-                font-size: 12px;
+                font-size: 8px;
```

Comment:
Font-Size reduction so our user it's able to actually read the Question in it's phone